plugins/callback/logstash.py

```diff
@@ -18,40 +18,79 @@
     options:
       server:
         description: Address of the Logstash server
-        env:
-          - name: LOGSTASH_SERVER
         ini:
           - section: callback_logstash
             key: server
-            version_added: 1.0.0
+        env:
+          - name: LOGSTASH_SERVER
         default: localhost
       port:
         description: Port on which logstash is listening
-        env:
-            - name: LOGSTASH_PORT
         ini:
           - section: callback_logstash
             key: port
-            version_added: 1.0.0
+        env:
+          - name: LOGSTASH_PORT
         default: 5000
+        version_added: 1.0.0
+      pre_command:
+        description: Executes command before run and result put to ansible_pre_command_output field.
+        ini:
+          - section: callback_logstash
+            key: pre_command
+        env:
+          - name: LOGSTASH_PRE_COMMAND
+        version_added: "1.0.0"
```

Comment:
If this is a new option, it needs to be
```suggestion
        version_added: "1.3.0"
```